lib/utils/stripHostname.js

```diff
@@ -0,0 +1,15 @@
+const { URL } = require('url');
+
+/**
+ * Remove everything except (hostname/ip):port pair
+ *
+ * @param {string} hostname
+ *
+ * @returns {string}
+ */
+function stripHostname(hostname) {
+  const url = new URL(hostname);
+  return url.host;
```

Comment:
I think we need a port too

Reply:
@shumkov `port` is a part of `host` property

Reply:
Wow! Didn’t know that.